CMakeLists.txt

```diff
@@ -180,10 +180,13 @@ if (MSVC)
    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 endif(MSVC)
 
+find_path(LIBURCU_INC urcu-bp.h)
```

Comment:
I think we need a urcu block, similar to the other blocks (although not optional).  Look at the USE_RPC_RDMA block as an example.  Probably something like
find_library()
find_path()
include_directories()

It should be here, rather than below with the existing find_library()

Reply:
Done, thanks for the review.